collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -171,16 +171,7 @@ public void setExternalDataManager(ExternalDataManager externalDataManager) {
     }
 
     public String getVersionedAppName() {
-        String versionName = "";
-        try {
-            versionName = getPackageManager()
-                    .getPackageInfo(getPackageName(), 0)
-                    .versionName;
-            versionName = " " + versionName.replaceFirst("-", "\n");
```

Comment:
@lognaturel tried and it's returning empty String now. So this seems to be working correctly without git.

What I did:
* Change the case statement in gradle config to prevent it matching my os (changed 'unix' to 'unixx').
* The `getVersionName` execute the catch section and return null.
* Check the BuildConfig class and see the VERSION_NAME showing "".
* Run the application and it's starts fine.

Good enough to allow user without git in their path to run the app?